src/main/java/core/basesyntax/FootballMatch.java

```diff
@@ -15,6 +15,7 @@
      * если не угадал ничего - возвращать 0.</p>
      */
     public int matchResult(int result1, int result2, int bet1, int bet2) {
-        return 0;
+        int answer = result1 == bet1 && result2 == bet2 ? 2 : result1 == bet1 || result2 == bet2 ? 1 : 0;
```

Comment:
this line is longer than 100 character https://mate-academy.github.io/style-guides/java/java.html#s4.4-column-limit